src/components/BurgerIngredients/IngredientCard/IngredientCard.jsx

```diff
@@ -5,22 +5,36 @@ import {
 } from "@ya.praktikum/react-developer-burger-ui-components";
 import styles from "./IngredientCard.module.css";
 import PropTypes from "prop-types";
+import { useDrag } from "react-dnd";
+import { ITEM_TYPES } from "../../../constants";
 
 const MemoCurrencyIcon = memo(CurrencyIcon);
 const MemoCounter = memo(Counter);
 
-const IngredientCard = ({ count, handleItemAddition, item, handleOpenIngredientDetailsModal }) => {
+const IngredientCard = ({ count, handleIngredientClick, item }) => {
```

Comment:
Очень хорошо, что Вы рассчитываете показания счётчика в редьюсере, а не в каких-либо компонентах. Это даёт возможность передавать в
компоненту IngredientCard только её показание счётчика count, а не весь массив выбранных элементов в бургере для расчёта этого показания.
Поэтому, я думаю, можно с пользой обернуть компоненту IngredientCard в HOC React.memo (https://praktikum.yandex.ru/learn/react/courses/b512c4ac-8797-4b96-935a-6b2865f27c9a/sprints/6021/topics/3daef531-37a0-4521-ac9c-dac879418335/lessons/1d29690a-33b0-4a1f-bac3-0967ddeedcc4/),
так как React.memo, решает нужно ли рендерить компоненту, проверяя изменились ли её текущие пропсы по сравнению с предыдыдущими, и рендерит компоненту только
в случае изменения, даже, если рендерится повторно родительская компонента.

Reply:
Компонент обернут в memo в самом низу в экспорте :)

```suggestion
export default memo(IngredientCard);
```

Reply:
Да, извините, я, ведь, это сначала и заметила, а потом переключила внимание на другое при проверке, и про это "memo" забыла. Но, думаю, есть какие-то отличия между memo и React.memo. Хорошо бы проэкспериментировать)